app/controllers/api/accounts_controller.rb

```diff
@@ -21,7 +21,7 @@ def find_accounts(accounts)
     if query_param.present?
       accounts = finder(accounts, query_param).apply
     elsif tag_param.present?
-      accounts.find_by(tag: tag_param)
+      accounts = accounts.find_by(tag: tag_param)
```

Comment:
? what does this have to do with username validation ?
move to sep. commit / branch.